game/client/c_baseentity.cpp

```diff
@@ -6489,8 +6492,14 @@ bool CBaseEntity::IsARagdoll()
 
 void CBaseEntity::SetMaterialOverride( const char* strMaterial ) 
 {
+	if (strMaterial == NULL) {
+		Q_memset(m_OverrideMaterial, 0, sizeof(m_OverrideMaterial));
+	} else {
+		Q_strncpy(m_OverrideMaterial, strMaterial, 255);
```

Comment:
why not use sizeof(m_OverrideMaterial) instead of hardcoding the length? would make it easier to adjust in the future